java-bootcamp-2018-tdd/src/test/java/com/globant/java_bootcamp_2018_tdd/RecentFileListTest.java

```diff
@@ -0,0 +1,86 @@
+package com.globant.java_bootcamp_2018_tdd;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Vector;
+
+import org.junit.Test;
+
+public class RecentFileListTest {
+
+	// For this tests the list of recent opened files have a size of 5, the
+	// predetermined(in the recent file list class) is 15
+	RecentFileList list = new RecentFileList(5);
+
+	@Test
+	public final void whenTheProgramIsRunForTheFirstTimeTheListIsEmpty() {

```

Comment:
add test to the start of the method. Also you can use a shorter name and add the full description as comments